### PR TITLE
Repaired broken link to documention for Setting Up Project in contributing.md

### LINKED ISSUE #619: Broken Link to Documentation for Setting Up Project
**Describe the bug**
The link to 'Documentation for Setting up the Project' is broken in CONTRIBUTING.md file.

**To Reproduce**
Clicking on the link in Contributing.md file should give return a 404 error.

**Expected behavior**
The link should redirect to the INSTALLATION.md file.

**Actual behavior**
The link instead redirects to a non-existent CONFIGURATION.md file.

**Additional details**
As per the discussion on this issue on organization's slack channel, I'd like to be assigned to solve this issue.

### PR DESCRIPTION
**Pull request by AbhishekM2001**
**What kind of change does this PR introduce?**

Repairs the broken link to Documentation to Set Up the Project in contributing.md file

**Issue Number:**

Fixes #619 

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,17 +5,19 @@ If you are new to contributing to open source, please read the Open Source Guide
...
-* [Ways to Contribute](#ways-to-contribute)
-* [Our Development Process](#our-development-process)
-    * [Issues](#issues)
-    * [Pull Requests](#pull-requests)
-    * [Branching Strategy](#branching-strategy)
-* [Contributing Code](#contributing-code)
-* Internships
-    * [GSoC](#gsoc)
-    * [GitHub Externship](#github-externship)
-* [Community](#community)
```
Comment (palisadoes) [inline CONTRIBUTING.md:18]: All the changes in the table of contents are unnecessary. Please revert this section.

**Review by palisadoes — CHANGES_REQUESTED** [review]
All the changes in the table of contents are unnecessary. Please revert this section.

File: CONTRIBUTING.md (hunk 1 above)
Comment (AbhishekM2001) [reply CONTRIBUTING.md:18]: OK, I'll make revert that change.

**Review by JamaicanFriedChicken — CHANGES_REQUESTED** [review]
You can also correct the spelling of the word testing in this line `- alpha-x.x.x: For stability teesting` of this file `CONTRIBUTING.md`. The spelling was corrected previously in your commit before reverting.

**Comment by AbhishekM2001** [comment]
The spelling has been corrected now.

**Comment by palisadoes** [comment]
Please merge with the latest updates. The code coverage error should go away.

**Review by JamaicanFriedChicken — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]

**Pull request merged**